spec/models/post_policy_spec.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe PostPolicy do
+  before do
+    SiteSetting.queue_jobs = false
```

Comment:
```suggestion
    Jobs.run_immediately!
```

This is sitesetting and is printing a deprecation warning: https://github.com/discourse/discourse/blob/ff93833fdfcf61d0277d7d91aadb9b97af9cba10/app/models/site_setting.rb#L88.

Reply:
Nice thanks!

Reply:
> This is sitesetting and is printing a deprecation warning:

Hahah I have really good grammar